platform/security/rest/security-rest-clientapi/src/main/java/org/codice/ddf/cxf/client/ClientFactoryBuilder.java

```diff
@@ -0,0 +1,189 @@
+/**
```

Comment:
❓ What value do we get from this being an interface? It doesn't hide any dependencies, and would need to change whenever the builder adds or removes arguments

Reply:
Yes, it would have to change, I couldn't see a way around that.